### PR TITLE
Create prescription drugs while seeding patients

### PR DESCRIPTION
**Pull request by govindkrjoshi**
**Story card:** [sc-8986](https://app.shortcut.com/simpledotorg/story/8986/validate-cphc-migration-script-in-staging)

## Because

We are currently not creating prescription drugs when seeding patients

## This addresses

Adds a prescription drugs seeder

### CODE REVIEW

File: lib/seed/runner.rb [hunk 1]
```diff
@@ -42,6 +42,9 @@ def call
       UserSeeder.call(config: config)
       seed_drug_stocks
 
+      ProtocolSeeder.call(config: config)
```
Comment (prabhanshuguptagit) [inline lib/seed/runner.rb:45]: This is already called via `FacilitySeeder`, is it needed?

File: lib/seed/prescription_drug_seeder.rb [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+        return {}
+      end
+
+      prescription_drugs = []
+      protocol_drugs = if facility.protocol.present?
+        facility.protocol.protocol_drugs
+      else
+        []
+      end
+      encounters.each_with_index do |encounter, index|
```
Comment (prabhanshuguptagit) [inline lib/seed/prescription_drug_seeder.rb:45]: We don't associate drugs to encounters anywhere currently. Shall we create drugs without encounters as the default, and let the user enable encounters for drugs through config if they want?

File: lib/seed/prescription_drug_seeder.rb [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+      @raw_to_clean_medicines = DrugLookup::RawToCleanMedicine.all
+      @logger.info "Starting #{self.class} with #{config.type} configuration"
+    end
+
+    def encounters
+      @facility.encounters
+    end
+
+    def prescription_drugs_to_create
+      (0...5).to_a.sample
```
Comment (prabhanshuguptagit) [inline lib/seed/prescription_drug_seeder.rb:30]: Let's pick this number from config akin to `MAX_BPS_TO_CREATE`.

**Review by prabhanshuguptagit — APPROVED** [review]
Looks great.

File: lib/seed/prescription_drug_seeder.rb (hunk 2 above)
Comment (govindkrjoshi) [reply lib/seed/prescription_drug_seeder.rb:45]: We are not creating encounters for prescription drugs, but creating prescription drugs on the day of the encounter.

**Pull request merged**